std/math/emulated/element.go

```diff
@@ -100,7 +91,7 @@ func (f *Field[T]) newInternalElement(limbs []frontend.Variable, overflow uint)
 // GnarkInitHook describes how to initialise the element.
 func (e *Element[T]) GnarkInitHook() {
```

Comment:
Maybe won't need GnarkInitHook anymore?

Reply:
I'm afraid we do, at least for convenience... so a circuit like 
```
type circuit struct {
A,B,C emulated.Element
}
```
if we remove gnark init hook, then before compiling, user has to do:
```
c := circuit{}
c.A = emulated.SomethingElementInit(... nb limbs ...) 
c.B = ...
// etc 
```

Reply:
Oh, you are right.